torch/fx/passes/_tensorify_python_scalars.py

```diff
@@ -256,6 +256,7 @@ def _sympy_interp(expr: sympy.Expr) -> MetaProxy:
                         isinstance(a, fx.Node)
                         and "val" in a.meta
                         and isinstance(zf := a.meta["val"], torch.SymFloat)
+                        and a.op == "placeholder"
```

Comment:
This seems safe, but I think we probably also want to eventually tensorify item() calls as well (and yes, it seems like we would need to do some careful dtype sleuthing in this case.)